AFQ/tasks/segmentation.py

```diff
@@ -61,6 +63,18 @@ def segment(data_imap, mapping_imap,
         trx = load_trx(streamlines, data_imap["dwi"])
         trx.streamlines._data = trx.streamlines._data.astype(np.float32)
         tg = trx.to_sft()
+    elif streamlines.endswith(".tck.gz"):
+        # uncompress tck.gz to a temporary tck:
+        temp_tck = streamlines.replace(".gz", "")
```

Comment:
I would make this temporary file using the Python tempfile library (https://docs.python.org/3/library/tempfile.html), so that it gets deleted as soon as you are done with it. Otherwise, you are duplicating that file.